zerver/lib/actions.py

```diff
@@ -1266,7 +1267,7 @@ def check_message(sender, client, message_type_name, message_to,
         send_pm_if_empty_stream(sender, stream, stream_name, realm)
 
         if stream is None:
-            raise JsonableError(_("Stream does not exist"))
+            raise JsonableError(_("Stream, %(stream_name)s, does not exist") % {'stream_name': cgi.escape(stream_name)})
```

Comment:
Should this be using the `quote=True` argument to `cgi.escape`?  May also want to look if there's a Django thing we should use instead.

Since displaying the stream name here is adding a new potential security issue (escaping done wrong here), we should add tests that contain bad characters and make sure the output is correct.